Mediastream & MediaStreamTrack | fix and move a sentence & update text (#31082)
mdn/content: files/en-us/web/api/mediastreamtrack/index.md

{{APIRef("Media Capture and Streams")}}

The **`MediaStreamTrack`** interface of the {{domxref("Media Capture and Streams API", "", "", "nocode")}} represents a single media track within a stream; typically, these are audio or video tracks, but other track types may exist as well.

> Some user agents subclass this interface to provide more precise information or functionality, like in {{domxref("CanvasCaptureMediaStreamTrack")}}.

Reviewer: ```suggestion
Some user agents subclass this interface to provide more precise information or functionality, such as {{domxref("CanvasCaptureMediaStreamTrack")}}.
```
